### ISSUE TITLE
Systematic: The window has crashed (reason:oom)

### ISSUE THREAD

**Issue by ClaudiuMec** [microsoft/vscode#113291]
- VSCode Version: 1.52.1
- OS Version: Windows_NT x64 10.0.18363

Steps to Reproduce:

1. Launch VSC (Explorer), waiting to populate my Box-Bundle
2. After some 6-7 minutes I get the error The window has crashed (reason: 'oom'). We are sorry for the inconvenience! You can reopen the window to continue where you left off

Does this issue occur when all extensions are disabled?: No
I only have the Azure Account 0.9.4 + Azure Storage 0.11.0 extensions
Without those I get no error, however I cannot access the blob  ("unable to resolve workspace folder")

**Comment by ravikkr18** [comment]
Getting Same error, while processing some big files in terminal.

**Comment by warriorwizard** [comment]
Mee too getting the same error

**Comment by joshuaxdmb** [comment]
Getting this error too while running XFoil in the terminal (several hundreds of times)

**Comment by kalba-security** [comment]
Getting the same issue repeatedly. It started right after I pasted a very long one-liner into the file I was working on.

**Comment by bpasero** [comment]
Can you try to reproduce with our nightly insider builds? You can give our preview releases a try from: https://code.visualstudio.com/insiders/

**Comment by changkaiyan** [comment]
When I try printing some lines, this bug occurs. I feel this is a buffer size error. I use AMD cpu.

**Comment by bpasero** [comment]
Closing due to inactivity.

**Comment by asphub** [comment]
Its happening continously

**Comment by lwm98** [comment]
I just met it when i running a vue project

**Comment by taijul169** [comment]
i am facing the same problem in my vscode editor when i work in node js project

**Comment by gustavodiazs** [comment]
Me too getting the same error

**Comment by mustafateke** [comment]
I got the same error Running Keras/Tensorflow, it always crashes after 1st epoch. I switched to Spyder and waiting the first epoch. 
GPU is about to 90%.

**Comment by StefanFirefly** [comment]
Got this too - continuous crash on restart attempts (C++ and Plain Python in the editor).  
Note: I was running Photoshop with a large image in parallel.  (VSCode only restarted when I closed down Photoshop)... Some memory/GPU allocation bug ??

**Comment by ericis** [comment]
I got this error with 18.6 GB of RAM To spare! (32 GB) Clearly VS Code isn't managing memory properly somewhere...

![image](https://user-images.githubusercontent.com/974919/107407906-43542400-6ad8-11eb-8a98-9f15236047b0.png)

**Comment by deepak1556** [comment]
Please provide the following details https://github.com/microsoft/vscode/issues/111112#issuecomment-776156417 using latest stable, and do open a separate issue so that its to isolate , debug per platform.

**Comment by mustafateke** [comment]
> I got this error with 18.6 GB of RAM To spare! (32 GB) Clearly VS Code isn't managing memory properly somewhere...
> 
> ![image](https://user-images.githubusercontent.com/974919/107407906-43542400-6ad8-11eb-8a98-9f15236047b0.png)
I think it is about GPU memory; Since it uses GPU hardware acceleration. You may disable it from its menu:

https://stackoverflow.com/questions/29966747/how-can-i-disable-gpu-rendering-in-visual-studio-code

**Comment by samratpodder** [comment]
Got the exact same error when running a  C Program. The terminal window got filled due to an infinite while loop and the window froze and then a window asked if I wanted to keep waiting/reopen/exit. When I choose to keep waiting that error popped up and reopen the window to the last recent edit.
